indicators/dependency_management.json

```diff
@@ -0,0 +1,25 @@
+{
+  "@context": "https://w3id.org/everse/rsqi#",
+  "@id": "https://w3id.org/everse/i/indicators/dependency_management",
+  "@type": "SoftwareQualityIndicator",
+  "name": "Software has dependency management solution",
+  "identifier": "https://w3id.org/everse/i/indicators/dependency_management",
+  "description": "Reviews how external libraries and dependencies are managed to ensure compatibility and security.",
+  "status": "Active",
+  "keywords": [
+    "sustaninability", "dependencies"
+  ],
+  "version": "1.0.0",
+  "contact": {
+    "@type": "Person",
+    "name": "Faruk Diblen"
+  },
+  "source": {
+    "@type": "Dataset",
+    "identifier": "https://zenodo.org/record/14852424",
+    "url": "https://zenodo.org/records/14852424/files/Software%20Quality%20Indicators.xlsx",
+    "name": "Software Quality Indicators from BioHackathon 2024 (Sheet: Sustanability, Row: Su7)"
```

Comment:
Now this one is interesting. According to that source, it comes from the RSMD guidelines, which state: 
```
Are the software dependencies described (RSMD-7.1)?
- Is the description of the dependencies available in a machine-actionable format?
```
Should we have 2 sources for this to point to the original source? https://fair-impact.github.io/RSMD-guidelines/8.rsmd_checklist/